packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1613,123 +1712,201 @@ class _TimePickerDialogState extends State<_TimePickerDialog> {
   }
 
   void _handleOk() {
+    if (_entryMode == TimePickerEntryMode.input) {
+      final FormState form = _formKey.currentState;
+      if (!form.validate()) {
+        setState(() => _autoValidate = true);
```

Comment:
prefer to reserve `=>` for one-liners that return a value
```dart
setState(() { _autoValidate = true });
```

Reply:
Done